frontera/contrib/backends/sqlalchemy/__init__.py

```diff
@@ -62,6 +63,9 @@ class State:
     state = Column(String(12))
     error = Column(String(20))
     meta = Column(PickleType())
+    headers = Column(PickleType())
+    cookies = Column(PickleType())
+    method = Column(String(6))
```

Comment:
I'm not sure if we need to pass PUT/DELETE (and other like OPTIONS or CONNECT) in sqlalchemy.